lib/baldr/grammar/version4010/set310.rb

```diff
@@ -35,13 +35,14 @@ def record_defs
           {id: 'M7', min: 0, max: 5}
         ]},
        {id: 'L0', min: 0, max: 120, level:[
-         {id: 'L5', min: 0, max: 9999},
-       ]},
+          {id: 'L5', min: 0, max: 9999},
+        ]},
+       {id: 'L1', min: 0, max: 20, level:[
+          {id: 'C3', min: 0, max: 1}
+        ]}
       ]},
       {id: 'L3', min: 1, max: 1},
-      {id: 'L1', min: 0, max: 20},
-      {id: 'C3', min: 0, max: 1},
-      {id: 'SE', min: 1, max: 1},
+      {id: 'SE', min: 1, max: 1}
     ]
```

Comment:
@midhunkrishna The substantive changes are good to go. One gripe. Spacing seems off between the N7 Loop and the L0 and L1. The latter 2 seem to be a one space to far to the left.

cc: @nathanielread , @scottkf